aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -493,24 +493,37 @@ public class AwsProperties implements Serializable {
    * AwsClientFactory} If set, all AWS clients will use this specified HTTP client. If not set,
    * {@link #HTTP_CLIENT_TYPE_DEFAULT} will be used. For specific types supported, see
    * HTTP_CLIENT_TYPE_* defined below.
+   *
+   * @deprecated will be removed in 1.4.0, use {@link org.apache.iceberg.aws.HttpClientProperties}
+   *     instead
    */
-  public static final String HTTP_CLIENT_TYPE = "http-client.type";
+  @Deprecated public static final String HTTP_CLIENT_TYPE = "http-client.type";
 
   /**
    * If this is set under {@link #HTTP_CLIENT_TYPE}, {@link
    * software.amazon.awssdk.http.urlconnection.UrlConnectionHttpClient} will be used as the HTTP
    * Client in {@link AwsClientFactory}
+   *
+   * @deprecated will be removed in 1.4.0, use {@link org.apache.iceberg.aws.HttpClientProperties}
+   *     instead
    */
-  public static final String HTTP_CLIENT_TYPE_URLCONNECTION = "urlconnection";
+  @Deprecated public static final String HTTP_CLIENT_TYPE_URLCONNECTION = "urlconnection";
 
   /**
    * If this is set under {@link #HTTP_CLIENT_TYPE}, {@link
    * software.amazon.awssdk.http.apache.ApacheHttpClient} will be used as the HTTP Client in {@link
    * AwsClientFactory}
+   *
+   * @deprecated will be removed in 1.4.0, use {@link org.apache.iceberg.aws.HttpClientProperties}
+   *     instead
    */
-  public static final String HTTP_CLIENT_TYPE_APACHE = "apache";
+  @Deprecated public static final String HTTP_CLIENT_TYPE_APACHE = "apache";
 
-  public static final String HTTP_CLIENT_TYPE_DEFAULT = HTTP_CLIENT_TYPE_APACHE;
+  /**
+   * @deprecated will be removed in 1.4.0, use {@link org.apache.iceberg.aws.HttpClientProperties}
+   *     instead
```

Comment:
The indentation for `instead` on the newline (and in all the other places) looks off to me, is this how spotless is formatting it?

Reply:
yup